plugins/inputs/redis/README.md

```diff
@@ -164,6 +164,10 @@ and the elapsed time since the last rdb save (rdb\_last\_save\_time\_elapsed).
     - lag(int, number)
     - offset(int, number)
 
+- redis_errorstat_total
+  - tags:
+    - err(int, number)
+
```

Comment:
Shouldn't this be
```suggestion
- redis_errorstat
  - tags:
    - err
  - fields:
    - total (int, number)
```
according to what you do in the code...

Reply:
There was only one field, so I wasn't sure how to handle it.
I think you're right about separating the tag and the field.